File: Solution_Practice_I/java/prob4_13.java
import java.util.Scanner;
public class prob4_13   {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int num, binacci=0, aux=1, contv=0, cont;
        do {
            System.out.println("Ingrese cuantos numeros ver:");
            num=input.nextInt();
        } while (num<0);
        while (num>0){
            contv+=1;
            for(int i=1; i<=contv; i++){
                if (num>0){
                    binacci+=aux;
                    aux=binacci-aux;
                    System.out.print(binacci+"!/"+contv);
                    num-=1;
                    if(num!=0){
                        System.out.print(" + ");
                    }
                }
            }
            
        }


        input.close();
    }
}

File: Solution_Practice_I/python/prob4_13.py
while True:
    num=int(input("Ingrese cuantos numeros ver: "))
    if num>=0:
        break
binacci=0
aux=1
contv=0
while num>0:
    contv+=1
    for i in range(1, contv+1):
        if num>0:
            binacci+=aux
            aux=binacci-aux
            print(f"{binacci}!/{contv}", end="")
            num-=1
            if num!=0:
                print(" + ", end="")
